libmisc/getdate.y

```diff
@@ -802,7 +809,7 @@ yylex (void)
 	  return LookupWord (buff);
 	}
       if (c != '(')
-	return *yyInput++;
+	return (unsigned char)*yyInput++;
```

Comment:
The function returns int.  Why this cast to unsigned char?

Reply:
This didn't make any sense. Sorry for including it. I've restored the previous version.